chia/full_node/mempool_manager.py

```diff
@@ -253,7 +254,30 @@ async def create_block_generator(
         lineage_cache = LineageInfoCache(self.get_unspent_lineage_info_for_puzzle_hash)
 
         return await self.mempool.create_block_generator(
-            lineage_cache.get_unspent_lineage_info, self.constants, self.peak.height
+            lineage_cache.get_unspent_lineage_info,
+            self.constants,
+            self.peak.height,
+            timeout,
+        )
+
+    async def create_block_generator2(
+        self,
+        last_tb_header_hash: bytes32,
+        timeout: float = 2.0,
```

Comment:
Are we going for 2 seconds instead of 1 second like the previous version?

Reply:
the original function doesn't have a timeout on CLVM serialization and compression. Since this includes that, it will take longer and it needs a longer timeout.
The original function can take as long as 2.5 seconds just for the serialization, so this is still a tightening of that timing.